cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Alonzo/Tx.hs

```diff
@@ -39,3 +68,107 @@ consumeSameBlock =
     assertBlockNoBackoff dbSync 1
   where
     testLabel = "consumeSameBlock-alonzo"
+
+consumeTxOutMigratedAlz :: IOManager -> [(Text, Text)] -> Assertion
+consumeTxOutMigratedAlz = do
```

Comment:
Probably I missed to mention, we try to do new test development only on the latest era (currently Babbage). We keep some tests from older eras just to be sure we didn't break something. So could these tests be migrated to the Babbage.hs module and using Babbage utilities? Eventually we want to have a seemless way to write tests for all eras, which will also reduce the cost of migrating the tests when a new era appears.

Reply:
ah yeah cool I just assumed they both needed them. I have these tests in Babbage era already as that's where I started them. I'll just remove theses as they are exactly the same really. You're right though we should probably have a way to write one test that works across multiple eras :)